jiant/utils/data_loaders.py

```diff
@@ -41,6 +41,9 @@ def load_span_data(tokenizer_name, file_name, label_fn=None, has_labels=True):
     rows = rows.apply(lambda x: realign_spans(x, tokenizer_name), axis=1)
     if has_labels is False:
         rows["label"] = False
+    else:
+        if label_fn is not None:
+            rows["label"] = rows["label"].apply(lambda x: label_fn(x))
```

Comment:
What about when `label_fn` is None? If you're doing nothing in that case, just collapse into elif

Reply:
Also, can we make labels 0 when there are no labels? More closely matches your usage with the label fn